src/jrd/version.h

```diff
@@ -0,0 +1,151 @@
+/*
+ *  The contents of this file are subject to the Initial
+ *  Developer's Public License Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the
+ *  License. You may obtain a copy of the License at
+ *  http://www.ibphoenix.com/main.nfs?a=ibphoenix&page=ibp_idpl.
+ *
+ *  Software distributed under the License is distributed AS IS,
+ *  WITHOUT WARRANTY OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing rights
+ *  and limitations under the License.
+ *
+ *  The Original Code was created by Vladyslav Khorsun
+ *  for the Firebird Open Source RDBMS project.
+ *
+ *  Copyright (c) 2019 Vladyslav Khorsun <hvlad@users.sourceforge.net>
+ *  and all contributors signed below.
+ *
+ *  All Rights Reserved.
+ *  Contributor(s): ______________________________________.
+ */
+
+#ifndef JRD_VERSION_H
+#define JRD_VERSION_H
+
+#define STRINGIZE_AUX(x)	#x
+#define STRINGIZE(x)		STRINGIZE_AUX(x)
+
+
+#ifdef RC_TARGET_engine13
+#define VER_FILEDESC "Engine plugin"
+
+#elif defined RC_TARGET_fb_lock_print
+#define VER_FILEDESC "Lock Print tool"
+
+#elif defined RC_TARGET_fbguard
+#define VER_FILEDESC "Guardian"
+
+#elif defined RC_TARGET_fbrmclib
+#define VER_FILEDESC "RM/COBOL client interface"
+
+#elif defined RC_TARGET_firebird
+#define VER_FILEDESC "Server main executable"
```

Comment:
Maybe just "Server executable"? I don' see any other "executable" in the descriptions.

Reply:
Ok